src/app/utils/__tests__/externalLinks.test.ts

```diff
@@ -17,6 +17,7 @@ onlyRunOnCI('externalLinks', () => {
       expect(Object.entries(linksGroup).length).toBeGreaterThan(0)
       // eslint-disable-next-line @typescript-eslint/no-unused-vars
       for (const [_linkName, url] of Object.entries(linksGroup)) {
+        if (Array.isArray(url)) continue // We don't need to go into faucets
```

Comment:
Isn't this a bit of over generalization?

Reply:
Please keep externalLinks in a testable format. That is the only purpose of the file.

Move the complex structure somewhere else and import the links into it for mapping

Reply:
Moved the faucet links to a separate file, and removed testing exceptions.

Reply:
But faucet link should be tested